src/SDKs/NotificationHubs/Management.NotificationHubs/Generated/HubsOperations.cs

```diff
@@ -0,0 +1,260 @@
+// Code generated by Microsoft (R) AutoRest Code Generator 1.0.1.0
```

Comment:
@amolr please add the right license header to all your code files.
Also please link the spec PR.
I suspect you are not using the right header constants in your swagger that is causing AutoRest to skip generating the right licensing header text.